esphome/components/online_image/online_image.h

```diff
@@ -160,6 +160,14 @@ class OnlineImage : public PollingComponent,
    * decoded images).
    */
   int buffer_height_;
+  /**
+   * The value of the Last-Modified HTTP header provided in the last response.
+   */
+  std::string last_modified_;
+  /**
+   * The value of the ETag HTTP header provided in the last response.
+   */
+  std::string etag_;
```

Comment:
The rest of the code has `etag` before `last_modified` ;-)
```suggestion
  /**
   * The value of the ETag HTTP header provided in the last response.
   */
  std::string etag_;
  /**
   * The value of the Last-Modified HTTP header provided in the last response.
   */
  std::string last_modified_;
```

Reply:
Also; should they be explicitly initialized to the empty string?

Reply:
I did the reordering and initialized them both to empty string.